tests/src/core/parse.js

```diff
@@ -43,6 +45,15 @@ describe('parse(content, { settings, ecmaFeatures })', function () {
     expect(parseSpy.args[0][1], 'custom parser to get parserOptions.filePath equal to the full path of the source file').to.have.property('filePath', path)
   })
 
+  it('passes with custom `parseForESLint` parser', function () {
+    const parseForESLintSpy = sinon.spy(eslintParser, 'parseForESLint')
+    const parseSpy = sinon.spy()
+    eslintParser.parse = parseSpy
```

Comment:
this can also be `const parseSpy = sinon.spy(eslintParser, 'parse');`

Reply:
> this can also be `const parseSpy = sinon.spy(eslintParser, 'parse');`

Nope, `eslintParser.parse` is `undefined`, `sinon` will throw an error.

See https://travis-ci.org/benmosher/eslint-plugin-import/builds/567436253?utm_source=github_status&utm_medium=notification

Reply:
ah, fair enough :-)